### PR TITLE
Smaller size docker image refactoring

### PR DESCRIPTION
**Pull request by sdurrheimer**
Like the prometheus one, here is my small size docker image refactoring.

promdash_optimized_docker_image

I got some hard times, ruby projects have so much dependencies :p

If some people could test it before merging this PR, I would really appreciate it. Just to be sure there is no regressions. I may have missed something.

@discordianfish @juliusv

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ The following is a quick-start example for running PromDash with a file-based lo
 
 First, create the SQLite3 database in a shared local Docker volume on the host:
 
-    docker run -p 3000:3000 -v /tmp/prom:/tmp/prom -e DATABASE_URL=sqlite3:/tmp/prom/file.sqlite3 prom/promdash ./bin/rake db:migrate
+    docker run -v /tmp/prom:/tmp/prom -e DATABASE_URL=sqlite3:/tmp/prom/file.sqlite3 prom/promdash ./bin/rake db:migrate
```
Comment (discordianfish) [inline README.md]: Why did you remove the portmapping?

**Comment by discordianfish** [comment]
Beside that comment, LGTM!

**Comment by stuartnelson3** [comment]
what is nodejs required for? to my knowledge it is only being used for the javascript tests?

**Comment by sdurrheimer** [comment]
@stuartnelson3 For the `rake assets:precompile` script. It needs a javascript runtime, I wasn't able to make it work with v8 gem.

**Comment by stuartnelson3** [comment]
K, thanks for the explanation  :+1:

**Comment by sdurrheimer** [comment]
But I try v8 gem again later. Removing nodejs will save some extra space.
